robot/framesystem/framesystem.go

```diff
@@ -154,6 +155,19 @@ func (svc *frameSystemService) Reconfigure(ctx context.Context, deps resource.De
 }
 
 // TransformPose will transform the pose of the requested poseInFrame to the desired frame in the robot's frame system.
+// NOTE: I don't understand the frame system service, how it works, how it is configured, what it has by default, etc.
```

Comment:
I think we should just meet and talk through all this. I even have a framesystem presentation somewhere.

Reply:
Agreed, I'll review any docs async you have before then to make best use of the time

Reply:
Search "Frame system" here: https://viam.atlassian.net/wiki/spaces/ENG/pages/15138817/Engineering+Product+Videos+Presentations+Talks